docs/apache-airflow/howto/custom-view-plugin.rst

```diff
@@ -24,12 +24,12 @@ core UI using the Plugin manager
 
 This is an example plugin for Airflow that displays absolutely nothing.
 
-In this plugin, two object reference are derived from the base class
+In this plugin, two objects reference are derived from the base class
```

Comment:
Should it be `two object references` instead?

Reply:
↑ Yes